hooks/useTBAPurchase.tsx

```diff
@@ -22,8 +26,9 @@ const useTBAPurchase = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
-  const purchase = async (quantity: number) => {
+  const purchase = async (quantity: number, hasWallet = null) => {
```

Comment:
why do we need to pass `hasWallet` as a param?

`useTbaPurchase` has access to `useUserProvider` as well right?

Reply:
fixed ✅